modules/virt-about-kubemacpool.adoc

```diff
@@ -15,6 +15,5 @@ Virtual machine instances created from that virtual machine retain the assigned
 KubeMacPool does not handle virtual machine instances created independently from a virtual machine.
 ====
 
-KubeMacPool is disabled by default.
-Enable a MAC address pool for a namespace by applying the KubeMacPool label to the namespace.
-
+KubeMacPool is enabled by default when you install {VirtProductName}.
+You can disable a MAC address pool for a namespace by removing the KubeMacPool label to the namespace. You can then re-enable it by adding the KubeMacPool label.
```

Comment:
```suggestion
You can disable a MAC address pool for a namespace by adding the KubeMacPool disabling label to the namespace. You can then re-enable it by removing this label.
```

Reply:
@Ram - Thanks for that. Have updated the docs.